requirements.txt

```diff
@@ -4,6 +4,7 @@
 pytest==7.2.1
 pytest-dotenv==0.5.2
 pytest-asyncio==0.21.0
+spacy[ja]==3.6.1
```

Comment:
Ideally, we don't want to add any new dependencies to the root package right now.

Is there a way to use NLTK for this? Otherwise, spacy should be imported in the function itself, wrapped by a try/except, similar to other 3rd party dependencies

Reply:
I moved the import into the method.